charts/hono/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2019 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+apiVersion: v1
+# Version of the Helm chart, by default, this will be the Maven project version
+version: 1.0.2
```

Comment:
FMPOV this should be > 1.0.2 (e.g. 1.0.3) as it is not the same as the 1.0.2 Helm Chart that we publish via Eclipse Download Servers.

Reply:
Yes I agree … what do you think about making this chart "2.0.0" … So we can make 1.0.x updates in Hono, and continue with 2.0.x in IoT Packages.

Reply:
Based on our discussion yesterday in the Hono community call, I'd actually prefer if we would simply rename the whole chart to e.g. `hono-test` so that it is clear to everybody that this is **not** the official Hono chart but we use this PR only to pave the way for the real chart which we will add in a later PR once the CI pipeline works reliably. WDYT?

Reply:
Why can't we just use (and update) this PR, and merge it once the CI gives us a green light? What is the benefit of naming it "test" first, and then creating a second PR?

Reply:
We cannot (should not) merge this PR because it doesn't contain the Hono Helm chart. The Hono Helm chart is (currently) maintained, released and hosted by the Hono project. This PR is *just for testing* and FMPOV that should also be reflected in its name so that nobody mistakes it *for the real thing*.
Once the CI pipeline works, we can start migrating the Hono Helm chart from the Hono project to the IoT Packages project. FMPOV that will be easiest, if we create a fresh PR at that point in time which contains exactly the code as taken from the Hono project.

Reply:
as a merge would publish the chart also on https://hub.helm.sh i would suggest to not merge it, if its not the real thing